src/components/v5/frame/ColonyHome/partials/ReputationChart/utils.ts

```diff
@@ -40,29 +47,45 @@ export const getTeamReputationChartData = (
       {
         id: rootTeam.id,
         label: rootTeam.metadata?.name || '',
-        value: Number(rootTeam.reputationPercentage),
+        value: 100,
         color: getTeamHexColor(rootTeam.metadata?.color),
       },
     ];
   }
 
+  /**
+   * Exclude the General team reputation from the domain reputation percentage computation
+   */
+  const normalisedTotalReputation = domainsWithoutRoot.reduce(
+    (reputation, team) => reputation + Number(team.reputation),
+    0,
+  );
+
   const topTeams = domainsWithoutRoot
+    // Filter out the domains without reputation in order to not display blank spaces in the chart
+    .filter((domain) => !!domain.reputationPercentage)
```

Comment:
@Nortsova I am wondering if this filter is not enough to omit teams having 0 reputation